python/src/equistore/labels.py

```diff
@@ -251,6 +251,48 @@ def asarray(self):
     def __contains__(self, label):
         return self.position(label) is not None
 
+    @staticmethod
+    def arange(name, *args):
+        """
+        A `Labels` instance with evenly spaced integers within a given interval.
+
+        The resulting `Labels` only contain one label, whose name
+        must be specified. The values of the labels depend on a set of
+        integer arguments, whose meaning is the same as those of the
+        `numpy.arange` function.
+        In particular:
+
+        :param `name`: Name of the resulting labels
+        :param `start`: (Optional) Lower bound of the range. If not specified,
+            it defaults to zero
+        :param `stop`: Upper bound of the range
+        :param `step`: (Optional) Spacing within the range of values. If not
+            specified, it defaults to one
```

Comment:
This documentation indicate that users can call this function with `Labels.arange(name="s", start=3, stop=4)` which is not the case. We could also have the function take `*args, **kwargs` and check both, I would first check how numpy handles this.

Reply:
I tried to check numpy's function arguments, but it seems to be written in C, so I gave up. I agree that using and checking kwargs is a good way to solve this. Probably also an example would help. I will work on this